src/tracing/tracker.py

```diff
@@ -273,7 +273,8 @@ def to_dict(self) -> Dict[str, Any]:
 class AbstractTracker(abc.ABC):
     """Abstract tracker class."""
 
-    _objects: Dict[int, Object] = dict()
```

Comment:
holy crap, so this made it into a static variable?

Reply:
yes